src/main/java/org/checkerframework/checker/androidresource/qual/ResourceTop.java

```diff
@@ -3,10 +3,7 @@
 import org.checkerframework.framework.qual.DefaultQualifierInHierarchy;
 import org.checkerframework.framework.qual.SubtypeOf;
 
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import java.lang.annotation.*;

```

Comment:
Can you undo all these wildcard import changes? In general explicit imports are less error prone.
It also makes it hard to see what actually changed in this PR.

Reply:
I will do.